container_service_extension/configure_cse.py

```diff
@@ -121,47 +145,45 @@ def check_cse_installation(config, msg_update_callback=utils.NullPrinter()):
                 msg_update_callback.error(msg)
                 SERVER_CLI_LOGGER.error(msg)
                 err_msgs.append(msg)
-        except Exception:  # TODO() replace raw exception with specific
```

Comment:
Not sure why do we need to remove catching generic Exception.
Although I am not a fan of catching generic Exceptions; would like to understand the reasoning here.